test/tree_diff_tests/mount.py

```diff
@@ -0,0 +1,66 @@
+import contextlib
+import os
+import re
+import subprocess
+import tempfile
+import time
+from enum import Enum
+
+from test.integration_tests.run import extract_image
+
+
+class ImageType(Enum):
+    QCOW2 = 0
+    TAR = 1
```

Comment:
We already have identifiers output types: `org.osbuild.qcow2` and `org.osbuild.tar`. Could we re-use those? I'd prefer a map from these names to functions which mount the image of the corresponding type (replacing the `switch` in `mount_image`).